### ISSUE TITLE
Charachter Jumping Mechanics

### ISSUE THREAD

**Issue by Stevemmm** [Game-Development-Club/7-Days-To-Nuke#11]
Hey guys, the character must be able to in the game. We can either go with the standard jump by having a `Physics.AddForce()` that will always be the same no matter how hard you press a key, or the jump will vary depending on how strong you press the jump key (Longer key press, higher jump). Please comment your opinions on which one we should go with! Someone also must take the task of creating this script, so comment if you want to create this script.

**Comment by jaxonRuiz** [comment]
I don't think we would need to make complicated jump mechanics, so we should probably make varying jump heights a lower priority.
